product_template_tags/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Product Template Tags",
     "summary": "This addon allow to add tags on products",
-    "version": "13.0.2.0.0",
+    "version": "13.0.2.1.0",
```

Comment:
you should not change version, the bot will do it. Also, this is a patch not a feature ;)